doc/man3/SSL_write.pod

```diff
@@ -28,6 +39,9 @@ SSL_write_ex2() functions similarly to SSL_write_ex() but can also accept
 optional flags which modify its behaviour. Calling SSL_write_ex2() with a
 I<flags> argument of 0 is exactly equivalent to calling SSL_write_ex().
 
+Each of these methods has a B<_writev> counterpart to accept vectored IO.
+The definition of the B<ossl_iovec> structure is included above.
```

Comment:
I think this deserves a bit more description around how users should call this. I don't think you can assume prior knowledge of how the UNIX/Linux iovec structure works.